src/components/collectionEditor/collectionEditor.js

```diff
@@ -67,23 +67,20 @@ import 'flexStyles';
 
     function addToCollection(apiClient, dlg, id) {
         const url = apiClient.getUrl(`Collections/${id}/Items`, {
-
             Ids: dlg.querySelector('.fldSelectedItemIds').value || ''
         });
 
-        apiClient.ajax({
-            type: 'POST',
-            url: url
-
-        }).then(() => {
+        apiClient.ajax({ type: 'POST', url: url })
+        .then(() => {
```

Comment:
```suggestion
        apiClient.ajax({ type: 'POST', url: url }).then(() => {
```

Reply:
You don't like this method?

Reply:
I find putting them on the next line more readable. It's easier to see at a glance where the different blocks are.

It's also what Prettier uses, so if we ever decide to use that (there has been talks about it before), it's one less change (We're already not very far from the style it uses).

Reply:
I agree, newline looks much cleaner when chaining